src/psyclone/tests/domain/gocean/transformations/gocean_extract_test.py

```diff
@@ -670,4 +657,5 @@ def test_change_prefix(monkeypatch):
     driver_name = "driver-main-update.f90"
     with open(str(driver_name), "r", encoding="utf-8") as driver_file:
         driver_code = driver_file.read()
-    assert "call NEW_psy_data%OpenRead('main', 'update')" in driver_code
+    assert ("call NEW_psy_data%OpenReadModuleRegion('main', 'update')"
```

Comment:
Would it make sense to add a test when DM is enabled in this file too or is it a base-class thing? Actually, even if it is a test would be good I think.

Reply:
I've added this.